sqlglot/dialects/sqlite.py

```diff
@@ -184,7 +185,9 @@ class Generator(generator.Generator):
             exp.ILike: no_ilike_sql,
             exp.JSONExtract: _json_extract_sql,
             exp.JSONExtractScalar: arrow_json_extract_sql,
-            exp.Levenshtein: rename_func("EDITDIST3"),
+            exp.Levenshtein: unsupported_args("ins_cost", "del_cost", "sub_cost", "max_dist")(
+                rename_func("EDITDIST3")
+            ),
```

Comment:
It seems that SQLite has a `langid` argument, we haven't accounted for that right? Looks like relatively low priority though, no need to expand the scope of this PR to include it

```
The editdist3() SQL function has 3 forms:

editdist3('TABLENAME');
editdist3('string1', 'string2');
editdist3('string1', 'string2', langid);
```

Reply:
Sounds good